tg_bot/modules/welcome.py

```diff
@@ -139,12 +141,31 @@ def new_member(bot: Bot, update: Update):
 
             if sent:
                 sql.set_clean_welcome(chat.id, sent.message_id)
-
+                
+        mute_user = sql.get_mute_pref(chat.id)
+        if mute_user:
+            try:
+                bot.restrict_chat_member(chat.id, new_mem.id,
+                                     can_send_messages=False,
+                                     can_send_media_messages=False,
+                                     can_send_other_messages=False,
+                                     can_add_web_page_previews=False)
+            except TelegramError:
+                pass
+            
+        return "<b>{}:</b>" \
+           "\n#NEW_USER" \
+           "\n<b>User:</b> {}" \
+           "\n<b>User ID:</b> <code>{}</code>".format(html.escape(chat.title),
+                                                        mention_html(new_mem.id, new_mem.first_name), new_mem.id)
+                
```

Comment:
Why is there so much whitespace here? PEP8 specifies 2 newlines between function définitions, you've got 4.